Fixing broken section URL in README.hb.md + formatting fixes

fisher6 opened the pull request.

fisher6 committed:
Fixing broken section URL in README.hb.md

fisher6 committed:
Spelling / formatting issues

fisher6 committed:
Pull request formatting

fisher6 committed:
Hopefully : will be in the right place now (RTL formatting sucks)

fisher6 commented:
@TomerPacific I tried and fixed some grammar/formatting issues in the README.hb.md. Can you please review this when you have a spare moment? 🙏

Roshanjossey commented on translations/README.hb.md:
```diff
@@ -214,14 +214,14 @@ git push origin <add-your-branch-name>
 </div>
 
 <div dir="rtl">
```
I'm planning to get rid of this section in all the tutorials. Could you please remove everything below this?

fisher6 replied:
Done

Roshanjossey commented on translations/README.hb.md:
```diff
@@ -187,15 +187,15 @@ git push origin <add-your-branch-name>
 עכשיו אפשר להתחיל לתרום לפרוייקטים אחרים. הכנו רשימה של פרוייקטים על נושאים קלים שאתם יכולים להתחיל לעבוד עליהם.
 
-<a href="https://roshanjossey.github.io/first-contributions/#project-list">רשימה של פרוייקטים ב-web app</a>
+<a href="https://roshanjossey.github.io/first-contributions/#project-list">רשימה של פרוייקטים ב-web app</a>.
```
first contributions has it's own org page now. could you please change this link to `https://firstcontributions.github.io/#project-list`?

TomerPacific replied:
@fisher6 , please go over the requested changes from @Roshanjossey

fisher6 replied:
Done

fisher6 committed:
Removing Self-Promotion section

As asked by @Roshanjossey [here](https://github.com/firstcontributions/first-contributions/pull/17976#discussion_r307333549)

fisher6 committed:
Modifying first contributions href to the updated URL

As asked by @Roshanjossey [here](https://github.com/firstcontributions/first-contributions/pull/17976/files#r307335858)

fisher6 commented:
Ready for rereview @Roshanjossey  🙃

Roshanjossey commented:
Thank you again @fisher6  and @TomerPacific
